Migration.Toolkit.Core/Mappers/CmsClassMapper.cs

```diff
@@ -288,130 +162,79 @@ private void MapClassDocumentTypeFields(CmsClass source, DataClassInfo target)
+                {
+                    case TcaDirective.ClearSettings:
                     {
                         field.Settings.Clear();
-                        field.SettingsMacroTable.Clear();
-                        field.DataType = FieldDataType.Assets;
-                        field.Settings[CLASS_FIELD_CONTROL_NAME] = FormComponents.AdminAssetSelectorComponent;
-                        field.Settings["MaximumAssets"] = "99";
-                        formInfo.UpdateFormField(columnName, field);
-                        break;
-                    }
-
-                    case Kx13FieldDataType.Guid:
-                    {
-                        field.DataType = FieldDataType.Guid;
-                        break;
-                    }
-
-                    case Kx13FieldDataType.Binary:
-                    {
-                        field.DataType = FieldDataType.Binary;
-                        break;
-                    }
-
-                    case Kx13FieldDataType.Xml:
-                    {
-                        field.DataType = FieldDataType.Xml;
                         break;
                     }
-
-                    case Kx13FieldDataType.Decimal:
+                    case TcaDirective.ClearMacroTable:
                     {
-                        field.DataType = FieldDataType.Decimal;
-                        break;
-                    }
-
-                    case Kx13FieldDataType.TimeSpan:
-                    {
-                        field.DataType = FieldDataType.TimeSpan;
+                        field.SettingsMacroTable.Clear();
                         break;
                     }
-
-                    case Kx13FieldDataType.Date:
+                    case TcaDirective.ConvertToAsset:
                     {
-                        field.DataType = FieldDataType.Date;
+                        // field.DataType = FieldDataType.Assets;
+                        field.Settings["MaximumAssets"] = "99"; // setting is missing in source instance, target instance requires it
                         break;
                     }
-
-                    case Kx13FieldDataType.DocRelationships:
+                    case TcaDirective.ConvertToPages:
                     {
-                        field.Settings.Clear();
-                        field.SettingsMacroTable.Clear();
-                        field.Settings[CLASS_FIELD_CONTROL_NAME] = FormComponents.AdminPageSelectorComponent;
-                        field.Settings["MaximumPages"] = "99";
-                        field.Settings["RootPath"] = "/";
-                        field.DataType = FieldDataType.Pages;
-                        field.Size = 0;
-                        formInfo.UpdateFormField(columnName, field);
+                        field.Settings["MaximumPages"] = "99"; // setting is missing in source instance, target instance requires it
+                        field.Settings["RootPath"] = "/"; // TODO tk: 2022-08-31 describe why?
```

Comment:
Please, specify this constant in a new file with other requirements for the target instance.